### ISSUE TITLE
[Search v2.4] [Auth] Reduce the limit of SearchForReports

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#49117]
Implement [this section](https://docs.google.com/document/d/1Uf7UwMYTmek2qWe37SqkQZnOZYrnrfPucOLzXMv-3WM/edit#bookmark=id.utwemy8708y4) of the doc 

<details><summary>Issue Owner</summary>Current Issue Owner: @luacmartins</details>

**Comment by Christinadobrzyn** [comment]
@luacmartins do you need me to add a C+ or is this going to be handled by eng only?

**Comment by luacmartins** [comment]
This is internal work only, so you can unassign yourself form this issue if you'd like

**Comment by luacmartins** [comment]
Got a draft PR up. This is still on hold for https://github.com/Expensify/App/issues/49122

**Comment by Christinadobrzyn** [comment]
just a heads up that I'm going to be ooo until 9/30. I'm not going to assign a BZ teammate since we're working on the PR/discussing the issue. If you need someone before Monday please reach out in Slack for a volunteer

**Comment by Christinadobrzyn** [comment]
I'm back from ooo - looks like we're still on hold for https://github.com/Expensify/App/issues/49122 so gonna keep this set to weekly

**Comment by Christinadobrzyn** [comment]
Hold for - https://github.com/Expensify/App/issues/49122

**Comment by Christinadobrzyn** [comment]
Ah sorry I just saw your message that I can unassign myself so I'll do that. Feel free to add me if you need anything related to BZ team - @luacmartins

**Comment by luacmartins** [comment]
Still pending deprecating Chat Finder

**Comment by luacmartins** [comment]
App PR is merged. We just need to wait on deploy to prod and we can continue work here

**Comment by luacmartins** [comment]
This is done.
